### DISCUSSION TITLE
mmcls绘制精度曲线报错

### DISCUSSION THREAD

**Discussion by huadouxiansheng** [open-mmlab/mmselfsup#720] [How-to] [answered]
我使用mmcls的版本是1.0.0rc3，
https://mmclassification.readthedocs.io/zh_CN/dev-1.x/useful_tools/log_result_analysis.html
按照上面参考想要绘制精度曲线
运行我自己的设置
`!python tools/analysis_tools/analyze_logs.py plot_curve work_dirs/resnet50_8xb128_coslr-100e_incus/20230320_201252/vis_data/20230320_201252.json --keys accuracy/top1 accuracy/top5  --legend top1 top5 --out results.jpg`
报错
Traceback (most recent call last):
  File "tools/analysis_tools/analyze_logs.py", line 9, in <module>
    from mmcls.utils import load_json_log
ImportError: cannot import name 'load_json_log' from 'mmcls.utils' (/root/mmclassification/mmcls/utils/__init__.py)
但是这个版本的包我没有改写，是版本有问题吗

**Comment by fangyixiao18** [comment, marked as answer]
you could pull the latest code of dev-1.x branch in mmcls, which supports `analyze_logs.py`
